packages/discord.js/src/util/Constants.js

```diff
@@ -70,6 +70,27 @@ exports.NonSystemMessageTypes = [
  * @typedef {TextChannel|NewsChannel|ThreadChannel|VoiceChannel|StageChannel} GuildTextBasedChannel
  */
 
+/**
+ * The types of channels that are text-based. The available types are:
```

Comment:
```suggestion
 * The types of guild channels that are text-based. The available types are:
```

to distinguish it from TextBasedChannelTypes